src/Symfony/Bundle/WebProfilerBundle/DependencyInjection/Configuration.php

```diff
@@ -39,6 +39,12 @@ public function getConfigTreeBuilder()
                 ->booleanNode('toolbar')->defaultFalse()->end()
                 ->scalarNode('position')
                     ->defaultValue('bottom')
+                    ->beforeNormalization()
+                        ->ifTrue(function ($v) { return null !== $v; })
+                        ->then(function () {
+                            @trigger_error('The "web_profiler.position" configuration key has been deprecated in Symfony 3.4 and it will be removed in 4.0.', E_USER_DEPRECATED);
```

Comment:
We can now use `setDeprecated()` instead, right?

Reply:
@javiereguiluz Can you make the change?